doc/Type/Any.pod6

```diff
@@ -1051,13 +1051,26 @@ Defined as:
     multi method reduce(Any:D: &with)
     multi sub reduce (&with, +list)
 
-Applying it to a class will always produce Nil.
-Applies its argument (or first argument, in case it's a sub) as an operator to
-all the elements in the object (or second argument), producing a single result.
-The argument must be an infix operator or take, in any case, two positional
-arguments.
+This routine combines the elements in a list-y object together, and thus
+produces a single result, by using a binary subroutine. The subroutine must be
+either an
+L<infix operator|https://docs.raku.org/language/operators#index-entry-infix_operator>
+or take two positional arguments.
 
-    (1..13).reduce( &[*] ).say; # OUTPUT: «6227020800␤»
+    say (1..4).reduce( &[+] );     # OUTPUT: «10␤»
+    say reduce &[+], 1..4;         # OUTPUT: «10␤»
+
+    say reduce &min, 1..4;         # OUTPUT: «1␤»
+
+    sub concat( Str() \s, Str() \t ) { s ~ t };
+    say reduce &concat, 'a'..'c';  # OUTPUT: «abc␤»
```

Comment:
Try to reveal intent, not syntax #1748. You're making a trivial synonym of ~. Just think about some binary sub that could be used as an example.